Sources/Cocoa/Extensions/UILabel+Extensions.swift

```diff
@@ -149,3 +149,32 @@ extension UILabel {
         )
     }
 }
+
+// MARK: Accessibility
+
+extension UILabel {
+    /// The accessibility trait value that the element takes based on the
+    /// text style it was given when created.
+    open override var accessibilityTraits: UIAccessibilityTraits {
+        get {
+            guard
+                let currentFont = font,
+                let textStyle = currentFont.fontDescriptor.object(forKey: .textStyle)
+                else {
+                    return super.accessibilityTraits
+            }
+
+            switch textStyle as! UIFont.TextStyle {
```

Comment:
I am sorry, can we do the same here? So use safe unwrap in guard instead of force unwrap in the switch?

Reply:
Also please fix indention for the cases in switch like:

```
switch value {
    case:
        return ""
}
``` 
not 

```
switch value {
case:
    return ""
}
```

Reply:
@mstana Thanks for that, I certainly didn't pay attention to it.